test/transaction/transaction.js

```diff
@@ -1274,7 +1274,7 @@ describe('Transaction', function() {
 });
 
 
-var tx_empty_hex = '01000000000000000000';
+var tx_empty_hex = '03000000000000000000';
```

Comment:
It's a test for the default transaction version. It has changed from 1 to 3, so that's just it. The actual test is at this line: https://github.com/dashevo/dashcore-lib/blob/master/test/transaction/transaction.js#L167